code/include/playfab/TaskQueue.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "XTaskQueue.h"
+
+namespace PlayFab
+{
+
+using AsyncWork = std::function<void()>;
+
+// RAII wrapper around XTaskQueueHandle
+class TaskQueue

```

Comment:
another utility class that I am presuming was brought over from somewhere else?

Reply:
Indeed, copied almost directly from XSAPI (namespace and other minor type changes)

---
In reply to: [595564025](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r595564025) [](ancestors = 595564025)